modules/cnv-resuming-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-maintenance-mode.adoc
+
+[id="cnv-resuming-node-maintenance-cli_{context}"]
+= Resuming a node from maintenance mode in the CLI
+
+Resume a node from maintenance mode and make it schedulable again by deleting 
+the `NodeMaintenance` object for the node. 
+
+.Procedure
+
+. Find the `NodeMaintenance` object in the `node-maintenance-operator` namespace:
++
+----
+$ oc get nodemaintenance -n node-maintenance-operator
```

Comment:
we don't have  `node-maintenance-operator` namespace, so don't need to use "-n node-maintenance-operator"